modular_skyrat/modules/GAGS/greyscale_configs.dm

```diff
@@ -785,6 +785,16 @@
 	name = "Worn Pentagram Strapped Dress"
 	icon_file = 'modular_skyrat/master_files/icons/mob/clothing/under/skirts_dresses.dmi'
 
+/datum/greyscale_config/plaidskirt_worn/teshari
+	name = "Worn Plaid Skirt - Teshari"
+	icon_file = 'modular_skyrat/modules/GAGS/icons/dressteshari.dmi'
+
+/datum/greyscale_config/sundress_worn/teshari
+	name = "Worn Sundress - Teshari"
+	icon_file = 'modular_skyrat/modules/GAGS/icons/dressteshari.dmi'
+	json_config = 'code/datums/greyscale/json_configs/sundress_worn.json'
```

Comment:
Line unnecessary as that comes with being a subtype of `/datum/greyscale_config/sundress_worn`
```suggestion
```